app/styles/components/buttons/button.css

```
@@ -11,30 +11,52 @@
-    grid-template-areas: "content spinner";
-    grid-template-columns: fit-content(100%) var(--button-spinner-width);
-    gap: var(--spacing);
-    align-items: center;
-    white-space: nowrap;
 
-    /* icon | text | spinner */
 
-    &>* {
-        grid-area: content;
+
+    /* Layout - reserve space for icon and spinner to prevent layout shift*/
```

Stylelint (17.4.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)